cmd/solana_exporter/utils.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+func assertf(condition bool, format string, args ...any) {
+	if !condition {
+		panic(fmt.Errorf(format, args...))
+	}
+}
+
+// prettyPrint is just a useful debugging function
+func prettyPrint(obj any) {
+	// For pretty-printed JSON, use json.MarshalIndent
+	prettyJSON, err := json.MarshalIndent(obj, "", "  ")
+	if err != nil {
+		fmt.Println("Error marshalling to pretty JSON:", err, ". obj: ", obj)
```

Comment:
Should this be using something like this? [klog.Info("error marshalling to pretty JSON: %s. obj: %v", err, obj)](https://pkg.go.dev/k8s.io/klog/v2#Infof)

Bare fmt.Print.. in code that is long running is generally a bug.